### PR TITLE
More detailed and correct(?) documentation for 'vtimer_set_msg' and 'timer_set_wakeup'.

### PR DESCRIPTION
**Pull request by noshky**
More detailed and correct(?) documentation for 'vtimer_set_msg' and 'timer_set_wakeup'.

### CODE REVIEW

**Comment by OlegHahm** [comment]
I think it's partly redundant to the IPC documentation, but in definitetely more helpful than the current version. Please rebase. ACK

File: sys/include/vtimer.h [hunk 1]
```diff
@@ -100,8 +100,17 @@ int vtimer_usleep(uint32_t us);
...
 /**
- * @brief   set a vtimer with msg event handler of type @ref MSG_TIMER
- * @param[in]   t           pointer to preinitialised vtimer_t
+ * @brief   Initialize a vtimer to send a message after a specified interval.
+ *
+ * Initialize a vtimer to send a message after a specified interval.
+ * The sent message will be constructed from the process id, the type and the
+ * pointer. The message will be sent from interrupt context.
+ * The message construction and sending is equivalent to:
+ * @code
```
Comment (authmillenon) [inline sys/include/vtimer.h]: would prefer

     * ~~~~~~~ {.c}
     * msg_t msg = { .type = type, .content = { .value = (unsigned int) ptr } };
     * msg_send_int(&msg, pid);
     * ~~~~~~~

instead (see http://www.stack.nl/~dimitri/doxygen/manual/markdown.html#md_fenced)

**Comment by noshky** [comment]
I've changed the documentation style as suggested.

**Comment by authmillenon** [comment]
Needs rebase.

**Comment by authmillenon** [comment]
(ACK upholds)

**Comment by LudwigOrtmann** [comment]
Please add module prefixes to the commit messages. (Compare the output of `git log`.)

**Comment by PeterKietzmann** [comment]
@noshky will you address the comments?

**Comment by PeterKietzmann** [comment]
+ rebase to current master

**Comment by OlegHahm** [comment]
Will be superseded by xtimer.

**Pull request closed without merging**